### PR TITLE
Update Microsoft.NETCore.CoreDisTools to version 1.0.1-prerelease-00005

### PR DESCRIPTION
**Pull request by zamont**
In order to work, this temporarily adds a direct reference to the win-x64 and win-x86 runtime packages. I'm not happy with this solution because I'd rather reference the identity package and use runtime.json to pull down the assets. I've started a thread offline to get help with the right solution.

### CODE REVIEW

**Comment by zamont** [comment]
@dotnet-bot test OSX10.12 x64 Check Innerloop Build and Test

**Comment by nattress** [comment]
I believe this is due to how the build system is building r2rdump. If I go into the r2rdump folder and run `dotnet publish --self-contained -r win-x64` the output publish folder grabs coredistools correctly. That's using the meta-package for CoreDisTools in r2rdump.csproj. I think you should keep the meta-package reference as below and look at why the rid isn't flowing to the project during build. This may be because coreclr doesn't really do managed projects much and we're missing some repo-level settings for this stuff.

```
<PackageReference Include="Microsoft.NETCore.CoreDisTools"> 
      <Version>1.0.1-prerelease-00005</Version>
</PackageReference>
```

File: src/tools/r2rdump/R2RDump.csproj [hunk 1]
```diff
@@ -14,11 +14,16 @@
 	<TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier>
 	<CLSCompliant>false</CLSCompliant>
 	<NoWarn>8002,NU1701</NoWarn>
```
Comment (nattress) [inline src/tools/r2rdump/R2RDump.csproj]: Nit: The indenting looks weird in this file. Would you mind converting any tabs to spaces so it lines up nicely.

**Comment by nattress** [comment]
I can run `r2rdump --disasm` fine with disassembly output working

**Comment by zamont** [comment]
@acmyu  I discovered one issue with the disassembler during my testing which I've created an issue for: #19564, but you can ignore this for demo purposes by disassembling an x64 R2R image. The new steps for building are
1. dotnet build --runtime win-x64
2. <CoreCLROutput>\netcoreapp2.0\win-x64\R2RDump.exe --in <input_file> --disasm

**Comment by zamont** [comment]
@acmyu @nattress can I get another quick review?

File: src/tools/r2rdump/R2RDump.cs [hunk 2]
```diff
@@ -405,7 +403,16 @@ private int Run(string[] args)
 
                     if (_disasm)
                     {
-                        disassembler = new Disassembler(r2r.Image, r2r.Machine);
+                        // TODO: Fix R2RDump issue where an x64 R2R image cannot be dissassembled with the x86 CoreDisTools
```
Comment (nattress) [inline src/tools/r2rdump/R2RDump.cs]: Could you please create a github issue for this (and stick the issue number in the comment) so we don't forget about it.

Comment (zamont) [reply src/tools/r2rdump/R2RDump.cs]: Issue is above #19564 but I can also put it in the code comment

Comment (zamont) [reply src/tools/r2rdump/R2RDump.cs]: Yup I can add it to the comment

**Pull request merged**